Google.PowerShell.IntegrationTests/SQL/SQL.GcSqlInstances.Tests.ps1

```diff
@@ -211,6 +211,29 @@ Describe "Import-GcSqlInstance" {
         { Import-GcSqlInstance "test-db2" "gs://gcsql-csharp-import-testing/testsql.gz" "newguestbook" } | Should Throw `
         "ERROR 1840 (HY000) at line 24: @@GLOBAL.GTID_PURGED can only be set when @@GLOBAL.GTID_EXECUTED is empty."
     }
+
+    It "should import a local file and delete the bucket for a local file upon completion" {
+        $oldBuckets = Get-GcsBucket
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\testsql.csv" "newguestbook" "entries" -UploadLocalFile} | Should not Throw
+        $newBuckets = Get-GcsBucket
+        $oldBuckets.Count | Should Be $newBuckets.Count
+    }
+
+    It "should delete the bucket for a local file upon a file error" {
+        $oldBuckets = Get-GcsBucket
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\filenotexist" "newguestbook" "entries" -UploadLocalFile} | Should Throw `
```

Comment:
Put the should Throw on the next line, which would allow you to avoid the line continuation character. (You can put a newline after the `|` operator.)